workflows/prognostic_c48_run/runtime/masking.py

```diff
@@ -0,0 +1,101 @@
+import xarray as xr
+from functools import partial
+from typing import (
+    Callable,
+    Hashable,
+    Mapping,
+    MutableMapping,
+    Optional,
+)
+
+from runtime.names import EAST_WIND, SPHUM, CLOUD
+
+
+__all__ = ["get_mask", "update_state_masked"]
+
+
+def update_state_masked(
```

Comment:
Switch this to something like `apply_mask`

Reply:
Renamed to `where_masked` since the underlying xarray function is named `where`.